crates/rpc/rpc-types/src/otterscan.rs

```diff
@@ -0,0 +1,72 @@
+use crate::{Block, Transaction, TransactionReceipt};
+use reth_primitives::{Address, Bytes};
+use serde::{Deserialize, Serialize};
+
+/// Operation type enum for `InternalOperation` struct
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub enum OperationType {
+    /// Operation Transfer
+    OpTransfer,
+    /// Operation Contract self destruct
+    OpSelfDestruct,
+    /// Operation Create
+    OpCreate,
+    /// Operation Create2
+    OpCreate2,
+}
+
+/// Custom struct for otterscan `getInternalOperations` RPC response
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct InternalOperation {
+    r#type: OperationType,
+    from: Address,
+    to: Address,
+    value: u128,
+}
+
+/// Custom struct for otterscan `traceTransaction` RPC response
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct TraceEntry {
+    r#type: String,
+    depth: u32,
+    from: Address,
+    to: Address,
+    value: u128,
+    input: Bytes,
+}
+
+/// Internal issuance struct for `BlockDetails` struct
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct InternalIssuance {
+    block_reward: String,
+    uncle_reward: String,
+    issuance: String,
```

Comment:
these are likely U256 I believe?

Reply:
Great question ! Well to get this one i had to read the Go code , 

```
type internalIssuance struct {
	BlockReward string `json:"blockReward,omitempty"`
	UncleReward string `json:"uncleReward,omitempty"`
	Issuance    string `json:"issuance,omitempty"`
}
```
this is the Go struct , and the values are the result of `EncodeBig`
```
	ret.BlockReward = hexutil.EncodeBig(minerReward.ToBig())
	ret.Issuance = hexutil.EncodeBig(issuance.ToBig())
	...
	ret.UncleReward = hexutil.EncodeBig(issuance.ToBig())
```

that from the documentation we get 
```
// EncodeBig encodes bigint as a hex string with 0x prefix.
```
I took the easy route of just reusing the Go type but i would love to know if it still makes sense to use a `U256` and have a custom serializer or if the default `U256` serializer behaviour is similar to the `EncodeBig`  output !

Reply:
> EncodeBig encodes bigint as a hex string with 0x prefix.

This is the same behaviour

Reply:
Great ! I'll change it over to U256 then , thanks for catching this !